src/webui/api/synccontroller.cpp

```diff
@@ -152,20 +152,22 @@ namespace
         syncData.clear();
 
         QVariantList removedItems;
-        foreach (QString key, data.keys()) {
+        for (auto i = data.cbegin(); i != data.cend(); ++i) {
+            const QString &key = i.key();
+            const QVariant &value = i.value();
```

Comment:
you are making a reference to a temporary object!
you should remove the reference.

Reply:
>you are making a reference to a temporary object!

You're wrong! See. declarations of these methods:
```c++
const Key &key() const;
const T &value() const;
```

Reply:
Ah, looked into the wrong document, my bad.